Introduction to Tecton/Tecton Tutorial: Introduction.py

```diff
@@ -122,8 +128,8 @@
 
 # COMMAND ----------
 
-training_data = fs.get_historical_features(spine=transactions, timestamp_key="timestamp").to_pandas()
-training_data.head(10)
+training_data = fs.get_historical_features(spine=transactions, timestamp_key="timestamp").to_spark()
+display(training_data.limit(10))
```

Comment:
depending on how this is used, might want to use `df.show()` instead of `display(df)` since display only works in databrick iirc

Reply:
Only used in Databricks, so this is fine.